lib/actions/index.coffee

```diff
@@ -35,3 +35,4 @@ module.exports =
 	build: require('./build')
 	deploy: require('./deploy')
 	util: require('./util')
+	preload: require('./preload')
```

Comment:
I forgot to add it.
It is here now.
I also added `resin-io/resin-preload#71-integrate-with-resin-cli` to `package.json`.